src/assets/effect.rs

```diff
@@ -46,4 +46,21 @@ impl EffectAssets {
             EffectType::Text(_) => None,
         }
     }
+
+    pub fn get_color(&self, effect_type: &EffectType) -> Color {
+        match effect_type {
+            EffectType::BarrierGlow => Color::rgb(1.0, 1.0, 1.4),
+            EffectType::AllyBlastExplosion => Color::rgb(4.0, 4.0, 4.0),
+            EffectType::EnemyBlastExplosion => Color::rgb(4.0, 4.0, 4.0),
+            EffectType::AllyBulletExplosion => Color::rgb(5.0, 5.0, 5.0),
```

Comment:
What does 5.0 mean here? The [bevy::prelude::Color::rgb](https://docs.rs/bevy/latest/bevy/render/color/enum.Color.html#method.rgb) docs say that the bounds are [0,1]

Reply:
Makes sense normally, since 0-1 is what's used for "base color".
Higher values than 1 are useless since you already have max for that color. For bloom, if you go above 1, the value is the intensity of the bloom, so 0 is no color, 1 is max color, 2 is max color + bloom, something like that (also depends on the base bloom settings)
Check one of [their examples](https://bevyengine.org/examples-webgpu/2D%20Rendering/bloom-2d/)